.circleci/config.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - restore_cache: *restore_yarn_cache
       - run: 
           name: Install dependencies
-          command: yarn install --frozen-lockfile --ignore-optional
```

Comment:
yarn install --ignore-optional does not work as expected. so removing optionalDependencies.